core/src/main/java/nl/nn/adapterframework/util/JdbcUtil.java

```diff
@@ -542,19 +542,17 @@ public static void fullClose(Connection connection, ResultSet rs) {
 			close(connection);
 			return;
 		}
-		Statement statement = null;
 		try {
-			statement = rs.getStatement();
+			if(!rs.isClosed()) {
+				try (Statement statement = rs.getStatement()) {
+					fullClose(connection, statement);
```

Comment:
Dit is niet nodig, want eind vd try {} closed het statement al.
Kan gewoon een leeg blok zijn dus.

En dan altijd close(connection) doen buiten de `if`.